src/app/upgrade.h

```diff
@@ -26,279 +26,12 @@
-
-    const QStringList backupFiles = backupFolderDir.entryList(
-                QStringList(QLatin1String("*.fastresume")), QDir::Files, QDir::Unsorted);
-    const QRegularExpression rx(QLatin1String("^([A-Fa-f0-9]{40})\\.fastresume$"));
-    for (const QString &backupFile : backupFiles) {
-        const QRegularExpressionMatch rxMatch = rx.match(backupFile);
-        if (rxMatch.hasMatch()) {
-            const QString hashStr = rxMatch.captured(1);
-            if (upgradeResumeFile(backupFolderDir.absoluteFilePath(backupFile), oldResumeData[hashStr].toHash()))
-                oldResumeData.remove(hashStr);
-            else
-                Logger::instance()->addMessage(QObject::tr("Couldn't migrate torrent with hash: %1").arg(hashStr), Log::WARNING);
-        }
-        else {
-            Logger::instance()->addMessage(QObject::tr("Couldn't migrate torrent. Invalid fastresume file name: %1").arg(backupFile), Log::WARNING);
-        }
-    }
-
-    for (auto i = oldResumeData.cbegin(); i != oldResumeData.cend(); ++i) {
-        const QVariantHash oldTorrent = i.value().toHash();
-        if (oldTorrent.value("is_magnet", false).toBool()) {
-            const QString &hash = i.key();
-            libtorrent::entry resumeData;
-            resumeData["qBt-magnetUri"] = oldTorrent.value("magnet_uri").toString().toStdString();
-            resumeData["qBt-paused"] = false;
-            resumeData["qBt-forced"] = false;
-
-            resumeData["qBt-savePath"] = oldTorrent.value("save_path").toString().toStdString();
-            resumeData["qBt-ratioLimit"] = static_cast<int>(oldTorrent.value("max_ratio", -2).toReal() * 1000);
-            resumeData["qBt-label"] = oldTorrent.value("label").toString().toStdString();
-            resumeData["qBt-name"] = oldTorrent.value("name").toString().toStdString();
-            resumeData["qBt-seedStatus"] = oldTorrent.value("seed").toBool();
-            resumeData["qBt-tempPathDisabled"] = false;
-
-            int queuePosition = oldTorrent.value("priority", 0).toInt();
-            resumeData["qBt-queuePosition"] = (queuePosition >= 0 ? queuePosition : 0);
-
-            QString filename = QString("%1.fastresume").arg(hash);
-            QString filepath = backupFolderDir.absoluteFilePath(filename);
-
-            QFile resumeFile(filepath);
-            QVector<char> out;
-            libtorrent::bencode(std::back_inserter(out), resumeData);
-            if (resumeFile.open(QIODevice::WriteOnly))
-                resumeFile.write(&out[0], out.size());
-        }
-    }
-
-    int counter = 0;
-    QString backupResumeFilename = oldResumeFilename + ".bak";
-    while (QFile::exists(backupResumeFilename)) {
-        ++counter;
-        backupResumeFilename = oldResumeFilename + ".bak" + QString::number(counter);
-    }
-    QFile::rename(oldResumeFilename, backupResumeFilename);
+    (void)ask; // to avoid including QtGlobal.h in this placeholder file just for calling Q_UNUSED()
```

Comment:
Isn't it better to remove (comment out) param name?

Reply:
>I have no preference. What do you say?

I suggested to comment out only the name of parameter.

Reply:
Done.